src/pytest_dbfixtures/factories/postgresql.py

```diff
@@ -116,10 +117,21 @@ def drop_postgresql_database(psycopg2, user, host, port, db):
     :param str host: postgresql host
     :param str port: postgresql port
     :param str db: database name
+    :param str version: postgresql version number
     """
     conn = psycopg2.connect(user=user, host=host, port=port)
     conn.set_isolation_level(psycopg2.extensions.ISOLATION_LEVEL_AUTOCOMMIT)
     cur = conn.cursor()
+    # We cannot drop the database while there are connections to it, so we
+    # terminate all connections first.
+    if float(version) >= 9.2:
```

Comment:
https://en.wikipedia.org/wiki/PostgreSQL#Release_history it should survive five years. It's not the first place in this code using float version comparisons.

I don't want to assume that PostgreSQL follows a Python or semver scheme. A proper solution would be to parse it into space-separated numbers and compare as tuples.